src/common/CMakeLists.txt

```diff
@@ -2,8 +2,6 @@ cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 
 set (sources
   math.cc
-  DepthCamera.cc
-)
+  DepthCamera.cc)
```

Comment:
Is that some code style standard for cmake or you just like it in a single line?I noticed you changed this in the entire project.